src/Button/deprecated/Button.test.jsx

```diff
@@ -11,9 +11,11 @@ describe('<Button />', () => {
   let button;
 
   beforeEach(() => {
+    const app = document.createElement('div');
+    document.body.appendChild(app);
     wrapper = mount(<Button
       {...defaultProps}
-    />);
+    />, { attachTo: app });
```

Comment:
Jest v25 uses jsdom v15 ([link](https://github.com/facebook/jest/pull/8851)) and all the tests related to `focus` started to fail due to the changes introduced in jsdom v15 ([link](https://github.com/jsdom/jsdom/issues/2924)). So I have updated the tests and introduced `attachTo` in order to fix this issue following these references: [1](https://github.com/enzymejs/enzyme/issues/2337#issuecomment-586583502) [2](https://ttfb.test.traveloka.com/testing-react-component-using-enzyme-jest-part-3/)

Reply:
added this to troubleshooting section of doc https://openedx.atlassian.net/wiki/spaces/AC/pages/3318054984/Node+16+Upgrade